emr/README.md

```diff
@@ -11,30 +18,111 @@ You will also need a `credentials` folder containing a `config` and `credentials
+```bash
+copying tests_aws/test_registry/test_data_sync.py -> dsgrid-0.1.0/tests_aws/test_registry
+copying tests_aws/test_registry/test_push_pull_online_mode.py -> dsgrid-0.1.0/tests_aws/test_registry
+copying tests_aws/test_registry/tests_s3_locks.py -> dsgrid-0.1.0/tests_aws/test_registry
+Writing dsgrid-0.1.0/setup.cfg
+Creating tar archive
+removing 'dsgrid-0.1.0' (and everything under it)
+Started a new cluster: j-2Y17R69QY2BKZ
+Cluster Status: STARTING - (no message)
+Cluster Status: STARTING - (no message)
+Cluster Status: STARTING - (no message)
+Cluster Status: BOOTSTRAPPING - Running bootstrap actions
+Cluster Status: BOOTSTRAPPING - Running bootstrap actions
+Cluster Status: BOOTSTRAPPING - Running bootstrap actions
+Cluster Status: BOOTSTRAPPING - Running bootstrap actions
+Cluster Status: BOOTSTRAPPING - Running bootstrap actions
+Cluster Status: BOOTSTRAPPING - Running bootstrap actions
+Cluster Status: BOOTSTRAPPING - Running bootstrap actions
+Cluster Status: BOOTSTRAPPING - Running bootstrap actions
+Cluster Status: BOOTSTRAPPING - Running bootstrap actions
+Cluster Status: BOOTSTRAPPING - Running bootstrap actions
+Cluster Status: BOOTSTRAPPING - Running bootstrap actions
+Cluster Status: BOOTSTRAPPING - Running bootstrap actions
+Cluster Status: BOOTSTRAPPING - Running bootstrap actions
+Cluster Status: BOOTSTRAPPING - Running bootstrap actions
+Cluster Status: BOOTSTRAPPING - Running bootstrap actions
+Cluster Status: BOOTSTRAPPING - Running bootstrap actions
+Cluster Status: BOOTSTRAPPING - Running bootstrap actions
+Cluster Status: WAITING - Cluster ready to run steps.
+Connecting to ec2-54-184-112-221.us-west-2.compute.amazonaws.com at 54.184.112.221
+Copying AWS config files
+Copying notebooks to master node, from: /home/ehale/dsgrid/dsgrid/notebooks...
+Opening tunnel to jupyter notebook server
 
+Jupyter Notebook URL: http://localhost:43431
+  Password is dsgrid
+  Press Ctrl+C to quit
 
+  To ssh into the master node: ssh -i /home/ehale/.ssh/ehale-dsgrid.pem hadoop@54.184.112.221
+```
+and based on this the user can login to Jupyter by putting `http://localhost:43431` in the address bar of her web browser or can ssh to the Master node via:
+```bash
+ssh -i /home/ehale/.ssh/ehale-dsgrid.pem hadoop@54.184.112.221
+```
+
+In either case, create a SparkSession attached to the cluster by entering this code block in a Jupyter cell or by running these lines in an ipython session or by including them in a script:
+```
+from pyspark.sql import SparkSession
+spark = SparkSession.builder.appName("app").getOrCreate()
+```
+
+### Shutting down
+
+If you are not working in the default Jupyter dsgrid_notebooks folder you might want to save your work back to your local machine before shutting the cluster down. 
```

Comment:
It is not clear to me how someone would actually do this; although I guess if you ssh into the master node you can rsync files back to your local machine? Is that the best advice we can give?